docs/integrations/fetching-incidents.md

```diff
@@ -98,35 +115,120 @@ incident = {
 
 ```python
 demisto.setLastRun({
     'start_time': timestamp_to_datestring(last_incident['time'])
 })
 ```
 
-When pulling incidents with ***fetch-incidents*** the ```setLastRun``` will not execute if there is an error with ***fetch-incidents***.
-
-### Sending the Incidents to Cortex XSOAR
-When all of the incidents have been created, we return the array of incidents by using the ```demisto.incidents()``` function. This is similar to the ```demisto.results()``` function, but is used exclusively to handle incident objects.
+### Send the Incidents to Cortex XSOAR
+When all of the incidents are created, the *demisto.incidents()* function returns an array of incidents in Cortex XSOAR.  
+This is similar to the *demisto.results()* function, but is used exclusively to handle incident objects.
 
-An example of it's usage is below:
+#### Example
 
 ```python
 # this command will create incidents in Cortex XSOAR
 demisto.incidents(incidents)
 ```
 
-If you do not have any incidents to return then just return an empty list to ```demisto.incidents()``` function.
+If there are no incidents to return,  *demisto.incidents()* returns an empty list.
 ```python
 # returning an empty list will keep the status as ok but no new incidents are created.
 demisto.incidents([])
 ```
 
+## Fetch Missing Incidents with Generic Lookback Methods
+This advanced feature uses generic lookback methods for fetching missing incidents.
+
+### Use case
+During a ***fetch-incidents*** run, some edge-case scenarios may cause missing incidents from the 3rd-party product.  
+The most common scenarios are:
+* Indexing issues in the product: For example, if incident A was created before incident B and only B was indexed, ***fetch-incidents*** will fetch only B and not A. If A was indexed after the fetch was called, the next fetch will fetch only from the created time of B.
+* An update in the incident information: Some implementations of ***fetch-incidents*** use a query filter to fetch only specific incidents. If initially an incident did not match the query (meaning, it was not fetched) but at some point was updated so that it now matches the query, ***fetch-incidents*** will not pull the updated incident because the time to fetch it already passed.
+
+### Solution
+The *look_back* parameter enables configuring how far back in time (in minutes) ***fetch-incidents*** will look to get the incidents that were created a while ago but indexed a few minutes ago.  
+In addition, the **LastRun** object stores the following fields to be used by the lookback methods:
+- **time** - The time to fetch the next fetch call (as in a regular fetch).
+- **limit** - The number of incidents retrieved in the next fetch. If the current fetch run has the same start_time as the last fetch (determined in **get_fetch_run_time_range()**), the limit will be increased by the limit parameter value, and then incidents retrieved in the last fetch will be filtered out.
```

Comment:
```suggestion
- **limit** - The maximal number of incidents retrieved in the next fetch. If the current fetch run has the same start_time as the last fetch (determined in **get_fetch_run_time_range()**), this field will be increased by the limit instance parameter value, and then incidents retrieved in the last fetch will be filtered out.
```
just to make it easier to follow (to avoid confusion between the last run field and the parameter)